README.md

```diff
@@ -306,3 +306,28 @@ sender.Close()
 
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## Releasing this SDK
+
+1. Make sure that all changes for the release are merged with the master branch
+2. Make sure that no applications are running that bind port 8080. See [Issue #102](https://github.com/wavefrontHQ/wavefront-sdk-go/issues/102)
+3. From the root directory of the project in the master branch, check that the following command run cleanly
+   1. `go install ./...`
+   2. `go test ./...`
+   3. `go vet ./...`
+4. Run `go mod tidy` to make sure it doesn't introduce new changes. If it brings new changes, check those in first before doing the release
+5. Find out the version of the last release of the SDK clicking on "Releases" in the about section on the right.
+6. Decide what the version of the next release will be. We follow [semantic versioning](https://semver.org/).
+7. Execute `git tag -a -m 'v0.10.3' 'v0.10.3'` replacing v0.10.3 with the new version.
+8. Push the tag up to github `git push upstream v0.10.3`
```

Comment:
Push the tag up to GitHub `git push upstream v0.10.3`

Reply:
Done.